tests/conftest.py

```diff
@@ -262,3 +266,26 @@ def some_method(self, test):
             }
         )
     build_file_tree(file_tree)
+
+
+def _containers_running(exec_path):
+    with local.cwd(exec_path):
+        if len(docker_compose("ps", "-a").splitlines()) > 0:
```

Comment:
You don't care about header lines

```suggestion
        if len(docker_compose("ps", "-aq").splitlines()) > 0:
```

Reply:
Yes, sorry, MB :man_facepalming: 
I meant to use `-q`, not `-a`. Although, now that I think of it, it makes sense to use `-aq` instead of only `-q`.
Should be fixed now, along with an improved error message.